src/main/java/uk/gov/justice/digital/delius/transformers/ConvictionTransformer.java

```diff
@@ -81,27 +85,28 @@ public ConvictionTransformer() {
     public static Conviction convictionOf(Event event) {
         final var courtAppearances = Optional.ofNullable(event.getCourtAppearances());
         return Conviction.builder()
-                .active(event.isActiveFlag())
-                .convictionDate(event.getConvictionDate())
-                .referralDate(event.getReferralDate())
-                .convictionId(event.getEventId())
-                .index(event.getEventNumber())
-                .offences(offencesOf(event))
-                .sentence(Optional.ofNullable(event.getDisposal())
-                    .map(disposal -> sentenceOf(disposal, event.getAdditionalSentences()))
-                    .orElse(null))
-                .custody(Optional
-                        .ofNullable(event.getDisposal())
-                        .flatMap(disposal -> Optional.ofNullable(disposal.getCustody()).map(ConvictionTransformer::custodyOf))
-                        .orElse(null))
-                .inBreach(event.isInBreach())
-                .failureToComplyCount(event.getFailureToComplyCount())
-                .breachEnd(event.getBreachEnd())
-                .latestCourtAppearanceOutcome(courtAppearances.map(ConvictionTransformer::latestOutcomeOf).orElse(null))
-                .responsibleCourt(Optional.ofNullable(event.getCourt()).map(CourtTransformer::courtOf).orElse(null))
-                .courtAppearance(courtAppearances.map(CourtAppearanceBasicTransformer::latestOrSentencingCourtAppearanceOf).orElse(null))
-                .awaitingPsr(isAwaitingPsr(event.getDisposal(), courtAppearances))
-                .build();
+            .orderManagers(orderManagersOf(event))
+            .active(event.isActiveFlag())
+            .convictionDate(event.getConvictionDate())
+            .referralDate(event.getReferralDate())
+            .convictionId(event.getEventId())
+            .index(event.getEventNumber())
+            .offences(offencesOf(event))
+            .sentence(Optional.ofNullable(event.getDisposal())
+                .map(disposal -> sentenceOf(disposal, event.getAdditionalSentences()))
+                .orElse(null))
+            .custody(Optional
+                .ofNullable(event.getDisposal())
+                .flatMap(disposal -> Optional.ofNullable(disposal.getCustody()).map(ConvictionTransformer::custodyOf))
+                .orElse(null))
+            .inBreach(event.isInBreach())
+            .failureToComplyCount(event.getFailureToComplyCount())
+            .breachEnd(event.getBreachEnd())
+            .latestCourtAppearanceOutcome(courtAppearances.map(ConvictionTransformer::latestOutcomeOf).orElse(null))
+            .responsibleCourt(Optional.ofNullable(event.getCourt()).map(CourtTransformer::courtOf).orElse(null))
+            .courtAppearance(courtAppearances.map(CourtAppearanceBasicTransformer::latestOrSentencingCourtAppearanceOf).orElse(null))
+            .awaitingPsr(isAwaitingPsr(event.getDisposal(), courtAppearances))
+            .build();
     }
```

Comment:
I think the whole file has had it's indentation reformatted - can we revert the whitespace changes so we can see the relevant code updates?

Reply:
no worries.